helix-core/src/textobject.rs

```diff
@@ -198,26 +198,26 @@ pub fn textobject_paragraph(
     Range::new(anchor, head)
 }
 
-pub fn textobject_surround(
+pub fn pair_surround(
     slice: RopeSlice,
     range: Range,
     textobject: TextObject,
     ch: char,
     count: usize,
 ) -> Range {
-    textobject_surround_impl(slice, range, textobject, Some(ch), count)
+    pair_surround_impl(slice, range, textobject, Some(ch), count)
 }
 
-pub fn textobject_surround_closest(
+pub fn pair_surround_closest(
```

Comment:
Shouldn't this be textobject_pair_ to be consistent with the rest? Also is it worth renaming this since it'll break user mappings

Reply:
I agree about the `textobject_pair_`

This shouldn't affect user mappings though, currently you can't bind `mim`/`mam` without macro bindings because of the `cx.on_next_key` in the `select_textobject` function from commands

Reply:
Prefix added. I can see the consistency argument. I just thought the original `textobject_surround` was misleading when reading the code, as my mind went to it being generic over any textobject.